packages/ui/primitives/signature-pad/signature-pad.tsx

```diff
@@ -214,7 +216,13 @@ export const SignaturePad = ({
   }, [defaultValue]);
 
   return (
-    <div className={cn('relative block', containerClassName)}>
+    <div
+      className={cn(
+        'relative block',
+        containerClassName,
+        isDisabled ? 'pointer-events-none opacity-50' : null,
+      )}
+    >
       <canvas
```

Comment:
You can do something like this to make it neater

```
      className={cn('relative block', containerClassName, {
        'pointer-events-none opacity-50': disabled,
      })}
```

Reply:
@dguyen Done

Reply:
Cheers!

Did you see the other note about using `disabled` instead of `isDisabled`?

Reply:
`disabled` matches how we disable inputs, buttons, fields.

`isSubmitting` is simply a variable we pass around.

Reply:
Sure, made the changes. Please review